modules/admin-ui-picard/app/src/thunks/seriesDetailsThunks.js

```diff
@@ -0,0 +1,250 @@
+
+        const response = await data.data;
+
+        // transform response for further use
+        let themeNames = transformToIdValueArray(response);
+
+        dispatch(loadSeriesDetailsThemeNamesSuccess(themeNames));
+    } catch (e) {
+        dispatch(loadSeriesDetailsThemeNamesFailure());
+    }
+}
+
+// update series with new metadata
+export const updateSeriesMetadata = (id, values) => async (dispatch, getState) => {
+    try {
+        let metadataInfos = getSeriesDetailsMetadata(getState());
+
+        let fields = [];
+        let updatedFields = [];
+
+        metadataInfos.fields.forEach(field => {
+            if (field.value !== values[field.id]) {
+                let updatedField = {
+                    ...field,
+                    value: values[field.id]
+                }
+                updatedFields.push(updatedField);
+                fields.push(updatedField);
+            } else {
+                fields.push({...field});
+            }
+        });
+
+        let data = new URLSearchParams();
+        data.append("metadata",JSON.stringify([{
+            flavor: metadataInfos.flavor,
+            title: metadataInfos.title,
+            fields: updatedFields
+        }]));
+
+        await axios.put(`/admin-ng/series/${id}/metadata`, data);
+
+        // updated metadata in series details redux store
+        let seriesMetadata = {
+            flavor: metadataInfos.flavor,
+            title: metadataInfos.title,
+            fields: fields
+        };
+        dispatch(loadSeriesDetailsMetadataSuccess(seriesMetadata));
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+export const updateSeriesAccess = (id, policies) => async (dispatch) => {
+    let data = new URLSearchParams();
+
+    let acls = {
+        acl: {
+            ace: policies
+        }
+    }
+    data.append("acl", JSON.stringify(acls));
+    data.append("override", true);
+
+    return axios.post(`admin-ng/series/${id}/access`, data, {
+        headers: {
+            'Content-Type': 'application/x-www-form-urlencoded'
+        }
+    })
+        .then(res => {
+            console.log(res);
+            dispatch(addNotification('info', 'SAVED_ACL_RULES'));
+            return true;
+        })
+        .catch(res => {
+            console.log(res);
+            dispatch(addNotification('error', 'ACL_NOT_SAVED'));
+            return false;
+        });
+}
+
+export const updateSeriesTheme = (id, values) => async (dispatch, getState) => {
+    let themeNames = getSeriesDetailsThemeNames(getState());
+
+    let themeId = themeNames.find(theme => theme.value === values.theme).id;
+
+    let data = new URLSearchParams();
+    data.append("themeId", themeId);
+
+    axios.put(`admin-ng/series/${id}/theme`, data)
+        .then(response => {
+            let themeResponse = response.data;
+
+            let seriesTheme = (transformToIdValueArray(themeResponse))[0].value;
+
+            dispatch(loadSeriesDetailsThemeSuccess(seriesTheme));
```

Comment:
See also comment in modules/admin-ui-picard/app/src/reducers/seriesDetailsReducers.js
Either add `dispatch(loadSeriesDetailsInProgress())` and `dispatch(loadSeriesDetailsFailure())` here, or don't change `isLoading` for this action in the reducer